foundry.toml

```diff
@@ -5,15 +5,15 @@
   block_timestamp = 1_680_220_800 # March 31, 2023 at 00:00 GMT
   bytecode_hash = "none"
   cbor_metadata = false
-  evm_version = "paris"
+  evm_version = "cancun"
```

Comment:
One question here: do you already know what chains these contracts will be deployed on?

Asking because `shanghai` has introduced `PUSH0`, which is not yet supported by all L2s.